server/models/migrations/v0.8.0-v0.9.0/setup-migration-script.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+# this script rebuild function and procedure 
+
+DATABASE="imck"
+FILENAME="migration"
+
+# path to the cleaned migration script
+CLEANED_MIGRATION_SCRIPT="./migrate.revised.sql"
+
+# path to bhima root from ./server/models/migrations/v0.8.0-v0.9.0
+BHIMA_PATH="$HOME/IMA/bhima-2.x"
```

Comment:
I propose we use default variables to allow us to set the BHIMA_PATH and DATABASE variables.  This will make it easier to work on multiple different environments.